config/betterquesting/resources/supersymmetry/lang/en_us.lang

```diff
@@ -1103,7 +1103,7 @@ susy.quest.db.522.desc=The §6CPU, or Central Processing Unit§r, serves as the
 susy.quest.db.523.title=RAM
 susy.quest.db.523.desc=RAM, or Random Access Memory, is essential for a computer's functioning. It provides fast, temporary storage for data actively used by the CPU. It is volatile memory that is lost when the computer is powered off. %n%nYou will need to insert §7RAM§r into your computer. Higher tiers of memory (RAM) mean §cmore memory available to the programs§r running on your computer. To run §cOpenOS§r, the §doperating system§r, you will want to use at least two §6tier 1 memory (RAM) sticks§r.%n%nThere are 6 tiers of memory, from 1 to 3.5.
 susy.quest.db.524.title=Analyzer
-susy.quest.db.524.desc=The analyzer is a handy tool for reading information about computers. Simply §a[Sneak Right Click]§r a block to get some information printed to the chat box. %nThis ranges from basic things like the address of components, to power levels in the subnetwork the block is in, and information on a crash, for example.%n%nRight now your computer will crash. Try using the Analyzer to see the error that causes the computer to not boot up: §cno bios found; install configured EEPROM§r.
+susy.quest.db.524.desc=The analyzer is a handy tool for reading information about computers. Simply §a[Sneak right-click]§r a block to get some information printed to the chat box. %nThis ranges from basic things like the address of components, to power levels in the subnetwork the block is in, and information on a crash, for example.%n%nRight now your computer will crash. Try using the Analyzer to see the error that causes the computer to not boot up: §cno bios found; install configured EEPROM§r.
```

Comment:
If you're uncapitalizing right-click everywhere, why not "sneak" as well?

Reply:
should be fixed